controller/user.controller.js

```diff
@@ -49,11 +46,6 @@ const getUserExperiment = (user) => {
   return user.experiments;
 };
 
-const setCookie = () => {
```

Comment:
why are you deleting this?

Reply:
I am not using this. I made this function and than used a different logic as the main flow progressed